config-overrides.js

```diff
@@ -0,0 +1,9 @@
+const ModuleScopePlugin = require("react-dev-utils/ModuleScopePlugin");
+
+module.exports = function override(config, env) {
```

Comment:
Codacy has a fix for the issue: [The variable `env` should be named `environment`. A more descriptive name will do too.](https://app.codacy.com/gh/devntv/useHooks_Apply_projects/pullRequest?prid=9710929)
```suggestion
module.exports = function override(config, environment) {
```